modules/component/src/let/let.directive.ts

```diff
@@ -105,8 +105,6 @@ export interface LetViewContext<PO> {
  */
 @Directive({ selector: '[ngrxLet]' })
 export class LetDirective<PO> implements OnInit, OnDestroy {
-  static ngTemplateGuard_ngrxLet: 'binding';
```

Comment:
Could/want we a test for this?

Reply:
It would require the "strictTemplates" setup for tests. AFAIK, that's not possible ("angularCompilerOptions" in `tsconfig.spec.json` doesn't affect tests).